Cartfile

```diff
@@ -1 +1,2 @@
 github "apple/swift-protobuf" ~> 1.0
+github "jrendel/SwiftKeychainWrapper"    ~> 3.2.0
```

Comment:
A friendly reminder to please document this new dependency in `./docs/dependency-management.md`.